FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -72,23 +72,19 @@ import Foundation
                                 uiDelegate: AuthUIDelegate? = nil,
```

Comment:
Why are the first two functions changed? They don't seem relevant to the recaptcha changes and running on the workQueue gets lost in the transition.

Reply:
Oh, didn't you mention that the objective C and swift functions should share an implementation? 
https://github.com/firebase/firebase-ios-sdk/pull/13192/files#r1697605692

Reply:
I mentioned that the implementation should be either in the async or callback version and be called by the other.